optaplanner-examples/src/main/resources/org/optaplanner/examples/vehiclerouting/optional/benchmark/vehicleRoutingStepLimitBenchmarkConfig.xml

```diff
@@ -26,7 +25,6 @@
   <solverBenchmark>
     <solver>
       <constructionHeuristic>
-        <constructionHeuristicType>FIRST_FIT_DECREASING</constructionHeuristicType>
```

Comment:
It is related. Using weakest/strongest/decreasing fit algorithms with a list variable is not possible at this point because it requires value/entity comparison and we haven't figured that out yet (https://issues.redhat.com/browse/PLANNER-2542). Until comparison is enabled, we have prohibited any CH configuration.